### DISCUSSION TITLE
Integration into home assistant maintaining the hue sync app

### DISCUSSION THREAD

**Discussion by Sergiodiaz53** [mvdwetering/huesyncbox#41] [General]
Hello,

I have a question regarding the integration of the hue sync into home assistant.

I have installed it using HACS but at some point I have to press the button and wait for a green light. If I do this, will my Hue syn app lose the connection to the sync box? My idea is to keep this working (hue sync box + hue sync app) but read some data using the integration to implement some scenes and other stuff.

Thanks in advance.

**Comment by mvdwetering** [comment]
Sorry, for the delay. Somehow I did not notice this message.

You can connect multiple applications to the Syncbox, so your official Hue Sync App will keep working even after linking with this integration. Not sure if there are limits on the amount of registered applications, but I never ran into an issue there.
